scrapy_poet/cache.py

```diff
@@ -0,0 +1,62 @@
+import abc
+import gzip
+import pickle
+import sqlite3
+
+import sqlitedict
+
+
+class _Cache(abc.ABC):
+
+    @abc.abstractmethod
+    def __getitem__(self, fingerprint: str):
+        pass
+
+    @abc.abstractmethod
+    def __setitem__(self, fingerprint: str, value) -> None:
+        pass
+
+    def close(self):
+        pass
+
+
+class SqlitedictCache(_Cache):
+
+    def __init__(self, path, *, compressed=True):
```

Comment:
@sortafreel This syntax forces the 2nd parameter to always use its **keyword**-value like this: `SqlitedictCache("path/to/somewhere", compressed=True)`.

Otherwise, calling it without the keyword args to the right the `*` specifier like `SqlitedictCache("path/to/somewhere", True)`, results in `TypeError: SqlitedictCache() takes 1 positional argument but 2 were given`.

Reference: https://www.python.org/dev/peps/pep-0570/#specification